src/web-component.js

```diff
@@ -34,13 +35,22 @@ class WebComponent extends HTMLElement {
   }
 
   static get observedAttributes() {
-    return ["code", "sense_hat_always_enabled", "step_number"];
+    return ["code", "sense_hat_always_enabled", "step_number", "steps"];
```

Comment:
I wonder whether we need to decide as a team what we're passing through to the editor from projects, since I think [Magda's PR](https://github.com/RaspberryPiFoundation/projects-ui/pull/2392) might be giving the web component an entire project